src/charm.py

```diff
@@ -43,6 +44,29 @@ def __init__(self, *args: typing.Any) -> None:
         super().__init__(*args)
         self.framework.observe(self.on.config_changed, self.reconciliation)
         self.framework.observe(self.on.spring_boot_app_pebble_ready, self.reconciliation)
+        self.ingress = IngressRequires(self, self._ingress_config())
+
+    def _ingress_config(self) -> typing.Dict[str, str]:
+        """Generate ingress configuration based on the Sprint Boot application and charm configs.
```

Comment:
`Spring` rather than `Sprint`?

Reply:
I think Niels fixed the "sprint" typo in his PR

Reply:
Perhaps merge those changes in so we are sure we don't unfix it again?

Reply:
I replaced all occurrences, thanks!